app/assets/stylesheets/govuk_publishing_components/components/_checkboxes.scss

```diff
@@ -1,10 +1,28 @@
 .govuk-checkboxes--nested {
   margin-left: (govuk-spacing(3) + 3px);
-  padding-left: govuk-spacing(4);
   box-sizing: border-box;
   border-left-style: solid;
   border-left-width: 4px;
   border-color: govuk-colour('grey-2');
+  margin-top: govuk-spacing(2);
```

Comment:
Might be good for maintainability to group some of these together i.e. put all the margin declarations together.